src/main/java/core/basesyntax/serviceimpl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String finalReport, File file) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file))) {
+            bufferedWriter.write(finalReport);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file");
```

Comment:
The catch block throws a RuntimeException, but it should also include the cause of the exception. This will provide more context for debugging purposes. Consider passing the caught exception 'e' to the RuntimeException constructor.